app.js

```diff
@@ -0,0 +1,142 @@
+const dh = document.getElementById('dh');
+const hamb = document.getElementById('hamb');
+const close = document.getElementById('close');
+const portfolio = document.getElementById('portfolio');
+const about = document.getElementById('about');
+const contact = document.getElementById('contact');
+
+hamb.addEventListener('click', () => {
+  dh.classList.toggle('open');
+});
+portfolio.addEventListener('click', () => {
+  dh.classList.toggle('open');
+});
+about.addEventListener('click', () => {
+  dh.classList.toggle('open');
+});
+contact.addEventListener('click', () => {
+  dh.classList.toggle('open');
+});
+close.addEventListener('click', () => {
+  dh.classList.toggle('open');
+});
+
+const desktop = document.querySelector('.desktop-menu');
+const links = `
+        <li id="portfolio">
+          <a href="#Portfolio">Portfolio</a>
+        </li>
+        <li id="about">
+          <a href="#About">About</a>
+        </li>
+        <li id="contact">
+          <a href="#Contact">Contact</a>
+        </li>
+        <li class="close" id="close">&times;</li>
+`;
+const ul = document.createElement('ul');
+ul.innerHTML = links;
+desktop.appendChild(ul);
+
+const storeData = [
+
+  {
+    name: 'Tonic',
+    description: 'A daily selection of privately personalized reads; no accounts or sign- ups required',
+    imagesrc: './image/Snapshoot Portfolio (1).svg',
+    technologies: ['html', 'css', 'javaScript'],
+    link: 'link_to_live_version.com',
+  },
+  {
+    name: 'Multi-post',
+    description: 'A daily selection of privately personalized reads; no accounts or sign- ups required',
+    imagesrc: './image/Snapshoot Portfolio (2).svg',
+    technologies: ['html', 'css', 'javaScript'],
+    link: 'link_to_live_version.com',
+  },
+  {
+    name: 'Multi-post',
+    description: 'A daily selection of privately personalized reads; no accounts or sign- ups required',
+    imagesrc: './image/Snapshoot Portfolio (3).svg',
+    technologies: ['html', 'css', 'javaScript'],
+    link: 'link_to_live_version.com',
+  },
+  {
+    name: 'Tonic',
+    description: 'A daily selection of privately personalized reads; no accounts or sign- ups required',
+    imagesrc: './image/Snapshoot Portfolio (4).svg',
+    technologies: ['html', 'css', 'javaScript'],
+    link: 'link_to_live_version.com',
+  },
+];
+
+const description = document.querySelectorAll('.work-paragraph');
+for (let i = 0; i < description.length; i += 2) {
+  description[i].innerHTML = storeData[i].description;
+  description[i + 1].innerHTML = storeData[i + 1].description;
+}
+
+// ..............popups...................
+
+const popups = document.querySelector('.popups');
+const popupProjectTitle = document.querySelector('.popup-project-title');
+const popupProjectDescription = document.querySelector('description');
+
+const disappearButton = document.querySelector('.popups-project-disapper-button');
+
+disappearButton.addEventListener('click', () => {
+  popups.style.display = 'none';
+});
```

Comment:
- [ ] [REQUIRED] As I have stated in the desktop popup, let's fix the styles which prevent a user from properly accessing the mobile popup. Kindly ensure that the popup fits well on the screen and that the popup is exactly as we have it on [template one](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1).

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="307" alt="Screenshot 2023-04-18 at 23 37 05" src="https://user-images.githubusercontent.com/58692590/232920254-c958ac5e-8d67-44de-b29f-0e17efbf88d2.png"> | <img width="275" alt="Screenshot 2023-04-18 at 23 33 35" src="https://user-images.githubusercontent.com/58692590/232919882-ce64a3f9-2757-4ad5-b544-2573447e8afe.png">